utilities/src/with_std/spmc.rs

```diff
@@ -61,6 +61,7 @@ impl<T> Sender<T> {
 	}
 }
 
+#[allow(dead_code)]
```

Comment:
?

Reply:
I meant to ask you about this - there is a `tokio::sync::watch::Receiver<()>` that is isn't explicitly used, so clippy flags it as dead code.

But I think it needs to remain there because the other end of the channel waits for it to be dropped (there's a doc comment a few lines above).

Reply:
Ill try a look after the meeting